### DISCUSSION TITLE
Plant Scanner

### DISCUSSION THREAD

**Discussion by Kaillera** [FakeFishGames/Barotrauma#4670] [Ideas and suggestions]
We can see minerals now, why not plants?

**Comment by DreamerOfRain** [comment]
Plants are always in caves - how about having each cave types being home to different plants, and one only need to see the cave opening to know what they can get instead of a scanner?

**Comment by hubismal** [comment]
imo it would be nice if the scanners showed the entrances to caves with loot in them or highlighted them or something. technically not a mining scanner but you'd think a tool designed for mining would be able to detect paths to potential veins of minerals
